src/find/mod.rs

```diff
@@ -182,18 +187,25 @@ fn do_find<'a>(args: &[&str], deps: &'a dyn Dependencies<'a>) -> Result<u64, Box
 
     let mut found_count: u64 = 0;
     let mut quit = false;
+    let mut has_some_process_error = false;
     for path in paths_and_matcher.paths {
         found_count += process_dir(
             &path,
             &paths_and_matcher.config,
             deps,
             &*paths_and_matcher.matcher,
             &mut quit,
+            &mut has_some_process_error,
         );
         if quit {
             break;
         }
     }
+
+    if has_some_process_error {
+        return Err("There are some process errors.".into());
+    }
```

Comment:
GNU `find` doesn't show an error after the processing, it only shows the errors you show on line 157.

Reply:
This code has been removed in the new commit. Thanks.